ICP/typescript-smart-contract-101/content/typescript-smart-contract-101.md

```diff
@@ -3,6 +3,8 @@
 
 Welcome to this ICP Azle Development 101 tutorial! This tutorial is designed to provide an introduction to developing on the Internet Computer Protocol (ICP) platform. In this guide, you will learn the basics of building and interacting with decentralized Azle canisters. By the end of this guide, you will have a solid understanding of developing for the ICP platform and be able to create the foundation for decentralized applications.
 
+If you want to skip the tutorial and jump straight into the code, you can access the final code for this tutorial in the [ICP-azle-boilerplate](https://github.com/dacadeorg/icp-message-board-contract) repository.
```

Comment:
It's not the boilerplate repo, it's a course repo.
The boilerplate repo is here: https://github.com/dacadeorg/ICP-azle-boilerplate.
However, we should keep the link to the course repo but change the title.